src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/view/TakesFlowPane.kt

```diff
@@ -0,0 +1,75 @@
+package org.wycliffeassociates.otter.jvm.app.ui.takemanagement.view
+
+import com.github.thomasnield.rxkotlinfx.toObservable
+import javafx.beans.property.SimpleObjectProperty
+import javafx.collections.ObservableList
+import javafx.geometry.Pos
+import javafx.scene.layout.FlowPane
+import javafx.scene.layout.Priority
+import javafx.scene.layout.VBox
+import org.wycliffeassociates.otter.common.data.workbook.Take
+import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.takecard.*
+import org.wycliffeassociates.otter.jvm.app.widgets.takecard.events.PlayOrPauseEvent
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import tornadofx.*
+import tornadofx.FX.Companion.messages
+
+class TakesFlowPane(
+    alternateTakes: ObservableList<Take>,
+    private val audioPlayer: () -> IAudioPlayer,
```

Comment:
audioPlayerProvider or audioPlayerBuilder... idk a lambda isn't an audio player and the name should reflect what this is

Reply:
Good point, although I'm working on this file now and I plan to replace that parameter and the next one with a lambda function called `createTakeCard` that just knows how to create the take card and doesn't care about the take card's details